doc/changes/latest.inc

```diff
@@ -17,7 +17,7 @@
 Current (0.23.dev0)
 -------------------
 
-.. |New Contributor| replace:: **New Contributor**
```

Comment:
please keep this line. Add a line for you but do not replace it

Reply:
Not sure why this is showing up as removed. It is kept in my latest commit.